docs/content/docs/2.components/slideover.md

```
@@ -197,6 +197,67 @@ slots:
+For `left` and `right` sides, the size controls the `max-width`. For `top` and `bottom` sides, it controls the `max-height`.
+::
+
+### Fullscreen
+
+Use the `fullscreen` prop to make the Slideover take the full width or height of the screen.
+
+::component-code
+---
+prettier: true
+ignore:
+  - title
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)